src/main/java/io/openapitools/swagger/JaxRSScanner.java

```diff
@@ -1,31 +1,60 @@
 package io.openapitools.swagger;
 
-import io.swagger.v3.oas.annotations.OpenAPIDefinition;
 import java.util.Collections;
 import java.util.HashSet;
 import java.util.Set;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
+
 import javax.ws.rs.Path;
+import javax.ws.rs.core.Application;
+
+import org.apache.maven.plugin.logging.Log;
 import org.reflections.Reflections;
 import org.reflections.scanners.ResourcesScanner;
 import org.reflections.scanners.SubTypesScanner;
 import org.reflections.scanners.TypeAnnotationsScanner;
 import org.reflections.util.ConfigurationBuilder;
 
+import io.swagger.v3.oas.annotations.OpenAPIDefinition;
+
 /**
  * Scan for classes with {@link Path} annotation or {@link OpenAPIDefinition}
- * annotation.
+ * annotation, and for {@link Application} instances.
  */
 class JaxRSScanner {
-    private Set<String> resourcePackages = Collections.emptySet();
 
-    private boolean useResourcePackagesChildren;
+	private final Log log;
```

Comment:
Ups - indentation seems wrong?
```suggestion
    private final Log log;
```

Reply:
My bad, should be fixed now.